e2e-tests/cypress/integration-with-acp/eligibility-check-decision-date.feature

```diff
@@ -0,0 +1,32 @@
+@wip
+Feature: Check that decision date is valid and eligible to commence an appeal with ACP integration
+  Note - This feature is intended to support redirection to ACP service as part of initial MVP
+
+  Scenario: (1) Decision date less than 12 weeks old can proceed
+    Given the application decision date is requested
+    When a decision date is provided that is no more than 12 weeks old
+    Then the appeal can be commenced
+
+  Scenario: (2) Decision date older than 12 weeks cannot proceed
```

Comment:
is it worth covering a decision date that is exactly 12 weeks old ?

Reply:
this appears covered with 84 days (?)

Reply:
Yes.